src/commands/music-commands/play.ts

```
@@ -22,97 +38,152 @@ export default {
+              name: "Enqueuing playlist:",
+              icon_url: harmonix.client.user.avatarURL,
+            },
             description: `${res.playlist.name} with ${res.tracks.length} tracks.`,
-            timestamp: new Date()
-          }
+            timestamp: new Date(),
+          },
         });
-      case 'SEARCH_RESULT':
+      case "SEARCH_RESULT":
         const max = Math.min(5, res.tracks.length);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)